packages/uniforms-antd/src/ListField.tsx

```diff
@@ -48,43 +49,57 @@ function List({
   wrapperCol,
   ...props
 }: ListFieldProps) {
+  const labelNode = !!label && (
+    <div>
+      {label}
+      {!!info && (
+        <span>
+          &nbsp;
+          <Tooltip title={info}>
+            <QuestionCircleOutlined />
+          </Tooltip>
+        </span>
+      )}
+    </div>
+  );
+
+  const borderStyle = {
+    ...(error ? { borderColor: 'rgb(255, 85, 0)' } : {}),
+    ...style,
+  };
+
   return (
     <div
       {...filterDOMProps(props)}
-      style={style}
+      style={borderStyle}
       className={classNames([className, 'ant-list', 'ant-list-bordered'])}
     >
-      {label && (
-        <div>
-          {label}
-          {!!info && (
-            <span>
-              &nbsp;
-              <Tooltip title={info}>
-                <QuestionCircleOutlined />
-              </Tooltip>
-            </span>
-          )}
-        </div>
-      )}
-
-      {!!(error && showInlineError) && <div>{errorMessage}</div>}
+      <Form.Item
+        label={labelNode}
+        hasFeedback
+        style={{ marginBottom: 0 }}
```

Comment:
This should be a constant, so it wouldn't change. Same goes for the `style` below.